projects/storefrontapp-e2e-cypress/cypress/integration/regression/wish-list.e2e-spec.ts

```diff
@@ -23,55 +27,79 @@ describe('Wish list', () => {
   });
 
   describe('Logged in', () => {
-    it('should add product to wish list from PDP', () => {
-      wishList.addToWishListFromPage();
-      wishList.verifyProductInWishListPdp();
-      wishList.verifyProductInWishList(wishList.products[0]);
+    beforeEach(() => {
+      cy.get('cx-login').click();
```

Comment:
This test case depends on the previous one. Visit the right page before add to whish list